### DISCUSSION TITLE
Add beachballs intro sequence

### DISCUSSION THREAD

**Discussion by april83c** [ppy/osu#24543] [Ideas]
Lots of people (including me) love this intro song from Stable, so maybe it should be added as a "intro sequence" option in Lazer?

**Comment by bdach** [comment]
https://github.com/ppy/osu/discussions/24403

**Reply by april83c** [reply]
this is talking about making seasonal intro sequences *actually work seasonally*, which is a thing that needs to be done, yes, but that's not what i'm asking for. what i'm asking for is to make *past* seasonal intro sequences available as options in the "intro sequence" dropdown in lazer

**Reply by april83c** [reply]
hi, just wanted to bump this

any chance we could get some past seasonal intro sequences (e.g. beachballs) as choices in this dropdown?

![image](https://github.com/user-attachments/assets/54a15553-ba9e-46ab-98c6-578a56d32dc2)

**Reply by april83c** [reply]
it would be so awesome if we had beachballs here...

**Reply by ziv-vy** [reply]
Also osumas intro sequence.
